interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
+<?php $collectThis=collectValidationPageRules("add_edit_issue",$_SERVER['PHP_SELF'])?>
```

Comment:
Wouldn't it make sense to declare that this file uses a validation to add a variable (ie. and also works as a flag for developers) that the top of this scripts (the first line of code after the gpl header) with the variable called something like $validator_id = "add_edit_issue";

This would then make it easier for developers to track and figure out which scripts are using this feature.